ansible/vagrant/Vagrantfile

```diff
@@ -6,6 +6,7 @@ require "yaml"
 ### RUN: vagrant plugin uninstall vagrant-openstack-plugin"
 ### Then RUN: "vagrant plugin install vagrant-openstack-provider"
 require 'vagrant-openstack-provider'
+require 'vagrant-hostmanager'
```

Comment:
Isn't that what https://github.com/kubernetes/contrib/blob/master/ansible/vagrant/Vagrantfile#L123 is all about? Doing this in a reproducible way across systems. Instead of doing some vagrant specific crap, like the salt stack is full of?